core/src/main/java/com/google/cloud/sql/core/DefaultConnectionInfoRepository.java

```diff
@@ -367,7 +374,7 @@ private Certificate fetchEphemeralCertificate(
     // Parse the certificate from the response.
     Certificate ephemeralCertificate;
     try {
-      ephemeralCertificate = createCertificate(response.getEphemeralCert().getCert());
+      ephemeralCertificate = createCertificate(response.getEphemeralCert().getCert()).get(0);
```

Comment:
Let's add a comment in the code about which cert `.get(0)` is gettiing...

Reply:
Done.